Project-Aurora/Project-Aurora/Devices/UnifiedHID/RoccatTyon.cs

```diff
@@ -0,0 +1,153 @@
+﻿using HidLibrary;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Aurora.Devices.UnifiedHID
+{
+
+    internal class RoccatTyon : UnifiedBase
+    {
+        private static HidDevice ctrl_device_leds;
+        private static HidDevice ctrl_device;
+
+        public RoccatTyon()
+        {
+            PrettyName = "Roccat Tyon";
+        }
+
+        private bool InitMouseColor()
+        {
+            return ctrl_device.WriteFeatureData(initPacket);
+        }
+
+        static bool WaitCtrlDevice()
+        {
+            for (int i = 1; i < 100; i++) // If still fails after 100 tries then timeout
+            {
+                // 150ms is the magic number here, should suffice on first try.
+                Thread.Sleep(150);
```

Comment:
I'm a bit concerned about this loop and the Thread sleeping, it could result in a significant delay to other devices if it ends up going through this loop many times. 

Side note: Defo need to make the Device connect functions async

Reply:
As said on discord, that code was on the other Roccat device.
I removed the thread sleeping and reduced the loop to 3 (removing the loop completely made it so the mouse was always stuck on the init colors)